chain/client/Cargo.toml

```diff
@@ -8,6 +8,7 @@ rust-version = "1.62.0"
 edition = "2021"
 
 [dependencies]
+anyhow = "1.0"
```

Comment:
This is a huge change -- we are adding a second way to handle errors to the client. We shouldn't be sneaking such things in unrelated PRs, please stick to the existing error handling strategy in the client for the time being, even if that strategy is not perfect: it's better to have one imperfect strategy than two.

Reply:
replaced anyhow with thiserror. LG?